app/src/components/ContactUs/SupportForm.jsx

```diff
@@ -49,73 +59,85 @@ class SupportForm extends Component {
       </section>);
     }
 
-    return (<section>
+    return (<section className={supportFormStyle['c-support-form']}>
       <h1>
         Support
       </h1>
       <form action="" method="POST" onSubmit={this.handleFormSubmit}>
-        <label htmlFor="name">Name</label>
-        <input
-          type="name"
-          id="support_name"
-          placeholder="Name"
-          required
-          onChange={this.handleChange}
-        />
-
-        <label htmlFor="email">Email</label>
-        <input
-          type="email"
-          id="support_email"
-          placeholder="email"
-          required
-          onChange={this.handleChange}
-        />
-
-        <label htmlFor="type">Type</label>
-        <div className={home.select_container}>
-          <select
-            id="support_type"
-            required
-            onChange={this.handleChange}
-          >
-            <option>Select an option...</option>
-            <option value="Error">Error</option>
-            <option value="Question">Question</option>
-            <option value="Feature">Feature</option>
-          </select>
+        <div className={supportFormStyle['contain-form']}>
+          <div className={supportFormStyle['container-inputs']}>
+            <label htmlFor="name">Name</label>
+            <input
+              className={supportFormStyle['input-text']}
+              type="name"
+              id="support_name"
+              placeholder="Name"
+              required
+              onChange={this.handleChange}
+            />
+
+            <label htmlFor="support_email">Email</label>
+            <input
+              className={supportFormStyle['input-text']}
+              type="Email"
+              id="support_email"
+              placeholder="email"
```

Comment:
Email (upper case E) please keep an eye on this kind of things. This is something that is easy for us to detect when testing after developing.